src/AirshipBindings.iOS/ApiDefinition.cs

```diff
@@ -269,8 +387,256 @@ interface UAirship
+		[Export("identifier")]
+		string Identifier { get; }
+
+		// @property (readonly, copy, nonatomic) NSArray<UANotificationAction *> * _Nonnull actions;
+		[Export("actions", ArgumentSemantic.Copy)]
+		UANotificationAction[] Actions { get; }
+
+		// @property (readonly, copy, nonatomic) NSArray<NSString *> * _Nonnull intentIdentifiers;
+		[Export("intentIdentifiers", ArgumentSemantic.Copy)]
+		string[] IntentIdentifiers { get; }
+
+		// @property (readonly, assign, nonatomic) UNNotificationCategoryOptions options;
+		[Export("options", ArgumentSemantic.Assign)]
+		UNNotificationCategoryOptions Options { get; }
+
+		// +(instancetype _Nonnull)categoryWithIdentifier:(NSString * _Nonnull)identifier actions:(NSArray<UANotificationAction *> * _Nonnull)actions intentIdentifiers:(NSArray<NSString *> * _Nonnull)intentIdentifiers options:(UNNotificationCategoryOptions)options;
+		[Static]
+		[Export("categoryWithIdentifier:actions:intentIdentifiers:options:")]
+		UANotificationCategory Category(string identifier, UANotificationAction[] actions, string[] intentIdentifiers, UNNotificationCategoryOptions options);
+	}
+
+	// @interface UANotificationContent : NSObject
+	[BaseType(typeof(NSObject))]
+	interface UANotificationContent
+	{
+		// @property (readonly, copy, nonatomic) NSString * _Nullable alertTitle;
+		[NullAllowed, Export("alertTitle")]
+		string AlertTitle { get; }
+
+		// @property (readonly, copy, nonatomic) NSString * _Nullable alertBody;
+		[NullAllowed, Export("alertBody")]
+		string AlertBody { get; }
+
+		// @property (readonly, copy, nonatomic) NSString * _Nullable sound;
+		[NullAllowed, Export("sound")]
+		string Sound { get; }
+
+		// @property (readonly, assign, nonatomic) NSNumber * _Nullable badge;
+		[NullAllowed, Export("badge", ArgumentSemantic.Assign)]
+		NSNumber Badge { get; }
+
+		// @property (readonly, nonatomic, strong) NSNumber * _Nullable contentAvailable;
+		[NullAllowed, Export("contentAvailable", ArgumentSemantic.Strong)]
+		NSNumber ContentAvailable { get; }
+
+		// @property (readonly, copy, nonatomic) NSString * _Nullable categoryIdentifier;
+		[NullAllowed, Export("categoryIdentifier")]
+		string CategoryIdentifier { get; }
+
+		// @property (readonly, copy, nonatomic) NSString * _Nullable launchImage;
+		[NullAllowed, Export("launchImage")]
+		string LaunchImage { get; }
+
+		// @property (readonly, copy, nonatomic) NSDictionary * _Nullable localizationKeys;
+		[NullAllowed, Export("localizationKeys", ArgumentSemantic.Copy)]
+		NSDictionary LocalizationKeys { get; }
+
+		// @property (readonly, copy, nonatomic) NSDictionary * _Nonnull notificationInfo;
+		[Export("notificationInfo", ArgumentSemantic.Copy)]
+		NSDictionary NotificationInfo { get; }
+
+		// @property (readonly, nonatomic, strong) UNNotification * _Nullable notification;
+		[NullAllowed, Export("notification", ArgumentSemantic.Strong)]
+		UNNotification UNNotification { get; }
```

Comment:
note: this property is simply "notification" in the original interface, but C# will not allow us to have that and the factory constructors below.